operator_ui/src/components/Table.ts

```diff
@@ -0,0 +1,29 @@
+import { createStyles, Theme } from '@material-ui/core/styles'
+
+// Contains styles to make a table row linkable
+//
+// This has been placed in the components folder because we ideally want to
+// wrap the link in a reusable components, however using Material UI 3
+// 'withStyles' leads to small freeze when navigating to the link, due to
+// the slowness of JSS. When we migrate to MUI 4/5 we can implement this with
+// makeStyles which doesn't have those problems
```

Comment:
You could also use a regular style objects instead of `createStyles`?

Reply:
I was attempting to attempting to customise the TableRow itself using this HOC technique

https://v3.material-ui.com/customization/overrides/#shorthand

However the problem can be found I found was that it spends too much time in JSS.

https://github.com/mui-org/material-ui/issues/10778#issuecomment-375905437

I could use styles to do it, but I'm hoping we can upgrade this to MUI v4 soon, and then make the improvement.